eZ/Publish/Core/Repository/UserService.php

```diff
@@ -1393,39 +1395,17 @@ protected function verifyPassword($login, $password, $spiUser)
      *
      * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException if the type is not recognized
      */
-    protected function createPasswordHash($login, $password, $site, $type)
+    public function createPasswordHash(string $password, int $hashType = APIUser::DEFAULT_PASSWORD_HASH): string
     {
-        $deprecationWarningFormat = 'Password hash type %s is deprecated since 6.13.';
-
-        switch ($type) {
-            case APIUser::PASSWORD_HASH_MD5_PASSWORD:
```

Comment:
This change should be mentioned in BC changes notes (`/doc/bc/changes-8.0.md`)

Reply:
I have even added action point for it like one week ago :D